### DISCUSSION TITLE
Viewing all issues in an organization

### DISCUSSION THREAD

**Discussion by MaGaroo** [community/community#67341] [Mobile] [answered]
I want to view all of the issues in some organizations (e.g. kubernetes & kubernetes-sigs), but it seems this feature is not available unless I'm a member of these organizations.
Also, I want to view issues the issues that I'm not involved in. But it seems I can only search for issues that I'm involved in.

My question is that is it possible at all? I know it's possible in Github web, but I couldn't do it in the android app.
Is there a specific reason that android app's search features are this much limited?

**Comment by stevepopovich** [comment, marked as answer]
hi @MaGaroo thanks for writing in. I am not sure of a screen/search on dotcom that lets you see all issues in an organization. 

In search, we chose to implement certain parts of search that we think most users will want. You can not use general search on mobile, but we can look into it. Thanks!

**Reply by MaGaroo** [reply]
Thanks Steve!
Just to note, we have https://github.com/issues and the search expression `is:issue user:kubernetes` will result in the issues in the Kubernetes organization.
